### PR TITLE
Add Additional Adafruit boards including the new Qualia ESP32-S3 and Camera ESP32-S3

### PR DESCRIPTION
**Pull request by makermelissa**
## Description of Change
Adds Additional Adafruit boards including the new Qualia ESP32-S3 and Camera ESP32-S3

## Tests scenarios
Please describe on what Hardware and Software combinations you have tested this Pull Request and how.

(*eg. I have tested my Pull Request on Arduino-esp32 core v2.0.2 with ESP32 and ESP32-S2 Board with this scenario*)

## Related links
Please provide links to related issue, PRs etc.

(*eg. Closes #number of issue*)

### CODE REVIEW

File: variants/adafruit_qualia_s3_rgb666/pins_arduino.h [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  TFT_G5 = 12,
+  TFT_B1 = 40,
+  TFT_B2 = 39,
+  TFT_B3 = 38,
+  TFT_B4 = 0,
+  TFT_B5 = 45,
+  TFT_PCLK = 1,
+  TFT_DE = 2,
+  TFT_HSYNC = 41,
+  TFT_VSYNC = 42;
```
Comment (P-R-O-C-H-Y) [inline variants/adafruit_qualia_s3_rgb666/pins_arduino.h:74]: Looks clearer and aligned to declare pins the same way as others. 
```suggestion
 static const uint8_t TFT_R1 = 11;
 static const uint8_t TFT_R2 = 10;
 static const uint8_t TFT_R3 = 9;
 static const uint8_t TFT_R4 = 46;
 static const uint8_t TFT_R5 = 3;
 static const uint8_t TFT_G0 = 48;
 static const uint8_t TFT_G1 = 47;
 static const uint8_t TFT_G2 = 21;
 static const uint8_t TFT_G3 = 14;
 static const uint8_t TFT_G4 = 13;
 static const uint8_t TFT_G5 = 12;
 static const uint8_t TFT_B1 = 40;
 static const uint8_t TFT_B2 = 39;
 static const uint8_t TFT_B3 = 38;
 static const uint8_t TFT_B4 = 0;
 static const uint8_t TFT_B5 = 45;
 static const uint8_t TFT_PCLK = 1;
 static const uint8_t TFT_DE = 2;
 static const uint8_t TFT_HSYNC = 41;
 static const uint8_t TFT_VSYNC = 42;
```

File: boards.txt [hunk 2]
```diff
@@ -11926,6 +11926,389 @@ adafruit_matrixportal_esp32s3.menu.EraseFlash.none.upload.erase_cmd=
...
+adafruit_camera_esp32s3.menu.DebugLevel.verbose=Verbose
+adafruit_camera_esp32s3.menu.DebugLevel.verbose.build.code_debug=5
+
+adafruit_camera_esp32s3.menu.EraseFlash.none=Disabled
+adafruit_camera_esp32s3.menu.EraseFlash.none.upload.erase_cmd=
+adafruit_camera_esp32s3.menu.EraseFlash.all=Enabled
+adafruit_camera_esp32s3.menu.EraseFlash.all.upload.erase_cmd=-e
+
+# Adafruit Qualia ESP32-S3 RGB666
+
```
Comment (P-R-O-C-H-Y) [inline boards.txt:12130]: Missing line of # to separate boards
```suggestion

##############################################################
# Adafruit Qualia ESP32-S3 RGB666

```

**Review by P-R-O-C-H-Y — CHANGES_REQUESTED** [review]
@makermelissa PTAL on my comments.

**Comment by P-R-O-C-H-Y** [comment]
@ladyada @makermelissa Can you please take a look on my comments quickly? We are preparing 2.0.14 release right now. Thanks

**Comment by ladyada** [comment]
lookin!

File: boards.txt (hunk 2 above)
Comment (ladyada) [reply boards.txt:12130]: done!

**Comment by ladyada** [comment]
ok i believe all were done. if i can have another 10 minutes i have another silly fix i have to make (i used OPI PSRAM pins on a board and turns out you cant do that :D )

**Comment by me-no-dev** [comment]
@ladyada you can have another 10 😆

File: boards.txt (hunk 2 above)
Comment (ladyada) [reply boards.txt]: hmm i see the change in the PR?

**Comment by ladyada** [comment]
(doing last hardware tests...)

File: boards.txt (hunk 2 above)
Comment (me-no-dev) [reply boards.txt]: yes, they showed up now... maybe GH is a bit slow

**Comment by ladyada** [comment]
ok done, for real! srsly! no backies

**Comment by me-no-dev** [comment]
You still got time till the CI is done ;)

**Comment by ladyada** [comment]
nah i think we're good now, lots of fun hardware and now it even works right ;)

**Comment by ladyada** [comment]
@me-no-dev woops this actually got reverted in a '5.1' update script

https://github.com/espressif/arduino-esp32/blame/master/variants/adafruit_metro_esp32s3/pins_arduino.h#L56

ill resubmit a new PR!

**Pull request merged**